.circleci/windows/vcvarsall.bat

```diff
@@ -0,0 +1,19 @@
+@echo on
```

Comment:
this helper script finds and executes `vcvarsall.bat` which is located on different places depending on the VS version and sets MSCV_VERSION var to the proper value.